src/tools/ergocub-fingers-tuning/README.md

```diff
@@ -47,4 +47,11 @@ Copy the `.csv` datasets created with `ident-finger` in the `data` folder. Then:
 1. Add to the path the folders `data` and `scripts`
 2. Open the live script `identification_tuning.mlx`
 3. Check the file names of the training and validation sets
+   - the file opened is the last one, unless the user run **init_ws** which resets the file name
+   - **init_ws** must also be run as the first command to insure the livescript works.
+   - it's better to also 
+   - it's better to also 'clear all output' by right-clicking in the livescript otherwise the report can be filled with old data and plots!
 4. Run it by pressing "Run" or F5
+   - sometimes (very often) the final command 'export' doesn't produce a complete pdf file. re-run this line by selecting it until the pdf looks fine 
```

Comment:
Uhm 🤔
This should be fixed upfront. Fine for now, but then the root cause ought to be sorted out.